cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java

```diff
@@ -286,6 +287,22 @@ public void testAuthorized() throws Exception {
   }
 
   @Test
+  public void testListPrivileges() throws Exception {
+    Set<Privilege> privileges = authorizer.listPrivileges(getUser("ns3_user1"));
+    Assert.assertTrue(privileges.contains(new Privilege(Authorizable.fromString("securekey:ns3.*_key-??"),
```

Comment:
It would be good to put all the expected privileges in a set and directly do a `Assert.assertEquals`. We'll get more information on a test failure this way.